lib/jar.ts

```diff
@@ -107,7 +107,7 @@ export function isJar(file: string): boolean {
 export function findJars(targetPath: string, recursive = false): string[] {
   const stats = fs.statSync(targetPath);
   const dir = stats.isFile() ? path.dirname(targetPath) : targetPath;
-  return glob.sync(`${dir}/${recursive ? '**/' : ''}*.jar`);
+  return glob.sync(`${dir}/${recursive ? '**/' : ''}*.@(j|w|a)ar`);
```

Comment:
According to [globs documentation](https://www.npmjs.com/package/glob#glob-primer)

Reply:
maybe change filename and function name, `archive.ts` and `findAchives` ?

Reply:
does isJar need to be same as this?

```
export function isJar(file: string): boolean {
  return !!file.match(/\.(([jwa]ar)|(zip))$/);
}
```

Reply:
in fact anything that mentions 'jar' in this file is better named archive?

Reply:
Done
I also added zip files to the glob since it's supported for one file in the `isJar` (now `isArchive`) function